doc/index.md

```diff
@@ -113,4 +113,41 @@ public class Connection
         var myResult = await rpc.InvokeWithParameterObjectAsync<string>("test/InvokeTestMethod");
     }
 }
+```
+
+## Crashing the process on exception
+In some cases, you may want to immediately crash the server and client processes if certain exceptions are thrown. In this case, overriding the `IsFatalException` method will give you the desired functionality. `IsFatalException` provides an exception filter through which you can access and respond to exceptions as they are caught.
```

Comment:
"...and client processes..." 

I don't think that's the scenario we're designing for, nor what you demonstrate below. It's just the server that crashes in this design, right?